src/dev/build/tasks/os_packages/docker_generator/templates/Dockerfile

```diff
@@ -51,7 +51,7 @@ EXPOSE 5601
 RUN for iter in {1..10}; do \
       {{packageManager}} update --setopt=tsflags=nodocs -y && \
       {{packageManager}} install --setopt=tsflags=nodocs -y \
-        fontconfig freetype shadow-utils libnss3.so {{#ubi}}findutils{{/ubi}} && \
+        fontconfig freetype shadow-utils nss {{#ubi}}findutils{{/ubi}} && \
```

Comment:
What effect does this have done the size of the container? What other packages does `nss` install that were not provided previously?

Reply:
It's minimal.  Installed size is 1.87mb.  I'll have to run a full build to compare, I think I'm getting different compression ratios than the release manager and my file size is quite a bit smaller currently

Reply:
List of included deps: https://centos.pkgs.org/8/centos-appstream-aarch64/nss-3.53.1-11.el8_2.aarch64.rpm.html